### DISCUSSION TITLE
3D Example apprent resistivity value not as expected

### DISCUSSION THREAD

**Discussion by GeorgeLiang3** [gimli-org/pyGIMLi#298] [Q&A]
In the [Example](https://www.pygimli.org/_examples_auto/3_dc_and_ip/plot_modTank3d.html#sphx-glr-examples-auto-3-dc-and-ip-plot-modtank3d-py) **3D modeling in a closed geometry**, if one change the resistivity for both facies to 10 
`res = [[1, 10.0], [2, 10.0]] .`
instead of 10 and 100 Ohm respectively to each facies, provided in the original example
`res = [[1, 10.0], [2, 100.0]]  # map markers 1 and 2 to 10 and 100 Ohmm, resp.`

The computed apparent resistivity is not the expected value 10. Is it a bug or did I do something run? 

Cheers,
George

**Comment by halbmy** [comment]
See answers and codes in #301. The long story short (all in detail described in Rücker et al. 2006):
1. The geometric factor needs to be computed numerically
2. All numerical computations have limited accuracy depending on mesh refinement at electrodes, mesh quality & shape functions.
3. By decreasing the cell size (or the refinement nodes or quadratic elements) one can improve accuracy and finally obtain the desired result.
In summary, ERT modelling is mainly a meshing issue due to the 1/r behaviour of the potential, unless secondary field approaches can be used.
